extra/meson.build

```diff
@@ -0,0 +1,27 @@
+desktop = 'linthesia.desktop'
+desktop_config = configuration_data()
+desktop_config.set('gettext', project_name)
+i18n.merge_file(
+  desktop,
```

Comment:
This is a hard error in the latest version of meson. merge_file() does not take a positional argument, and since 0.60 has type-checked this and errored out when incorrect arguments are supplied. In 0.60.1, it was changed to a deprecation (because it turned out many people were doing this incorrect thing) to give people time to fix it, and now in 0.61 it is an error again.

Reply:
what is the suggested change?

Reply:
Ah remove that, ok